app/users/shared/edit-student-details/edit-student-details.component.html

```diff
@@ -1,4 +1,4 @@
-<md-card *ngIf="user.role === 'Student'">
+<md-card *ngIf="user.roles === 'Student'">
```

Comment:
I think this won't work if the user has multiple roles, e.g. `[ "Student", "Guest" ]`